packages/@aws-cdk/cdk-common/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@aws-cdk/cdk-common",
```

Comment:
The `cdk-` prefix is a bit misleading because one might think this is related to the core library, but in fact it's related to the toolchain. Can we rename this to `@aws-cdk/toolchain-utils` or `@aws-cdk/toolchain-common`